docusaurus/docs/tutorials/build-an-app-plugin.md

```diff
@@ -0,0 +1,186 @@
+---
+id: build-an-app-plugin
+title: Build an app plugin
+sidebar_position: 10
+description: Learn how to create an app plugin.
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - app
+---
+
+import CreatePlugin from '@shared/create-plugin-frontend.md';
+import PluginAnatomy from '@shared/plugin-anatomy.md';
+
+## Introduction
+
+In this tutorial you will learn how to create _app plugins_. App plugins are Grafana plugins that allow you to bundle data sources and panel plugins within a single package. They also enable you to create custom pages within Grafana, providing a dedicated space for documentation, sign-up forms, or integration with other services via HTTP.
+
+App plugins are displayed in the navigation menu and offer the flexibility to define [UI Extensions](../ui-extensions/).
+
+### Prerequisites
+
+- Grafana v9.0 or later
+- [LTS](https://nodejs.dev/en/about/releases/) version of Node.js
+
+## Create a new app plugin
+
+<CreatePlugin pluginType="app" />
+
+:::note
+
+If you choose to have a backend for your app plugin, you must run `mage -v` before starting Grafana with Docker.
+
+:::
+
+## Anatomy of a plugin
+
+<PluginAnatomy />
+
+## Development workflow
+
+Next, you'll learn the basic workflow of making a change to your app, building it, and reloading Grafana to reflect the changes you made.
+
+The first step is to add your panel to a dashboard:
+
+1. Open Grafana in your browser.
+1. Go to **Apps** -> **Your App Name**.
+
+Now that you can view your app root page (page one in the example), try making a change to the app plugin:
+
+1. In `PageOne.tsx`, change some of the page text content:
+
+   ```tsx title="src/pages/PageOne.tsx"
+   <PluginPage>New page content</PluginPage>
+   ```
+
+1. Save the file.
+1. Reload Grafana. Your changes should appear in the browser.
```

Comment:
> Reload Grafana
Does this mean refresh the browser, or `docker compose down && docker compose up --build`?

Reply:
Good point!